packages/web-api/src/types/response/SlackListsAccessDeleteResponse.ts

```diff
@@ -0,0 +1,10 @@
+import type { WebAPICallResult } from '../../WebClient';
+
+export type SlackListsAccessDeleteResponse = WebAPICallResult & {
+  channel?: string;
+  error?: string;
+  needed?: string;
+  ok?: boolean;
+  provided?: string;
+  ts?: string;
+};
```

Comment:
```suggestion
export type SlackListsAccessDeleteResponse = WebAPICallResult & {
  error?: string;
  needed?: string;
  ok?: boolean;
  provided?: string;
};
```

👁️‍🗨️ question: I haven't tested this but AFAICT this response contains just `ok` but not `channel` or `ts`?

☕ suggestion: We can perhaps glean common values from actual responses or attempt the [autogenerate](https://github.com/slackapi/node-slack-sdk/blob/main/scripts/generate-web-api-types.sh) script after landing https://github.com/slackapi/java-slack-sdk/pull/1537!